feat: add protocolbuffers/protobuf/protoc

mikutas opened the pull request:
#1135 #6998 [protocolbuffers/protobuf/protoc](https://github.com/protocolbuffers/protobuf): Protocol Buffers - Google's data interchange format

```console
$ aqua g -i protocolbuffers/protobuf/protoc
```

## How to confirm if this package works well

Reviewers aren't necessarily familiar with this package, so please describe how to confirm if this package works well. Please confirm if this package works well yourself as much as possible.

Command and output

```console
$ protoc --version
libprotoc 3.21.7
```

If files such as configuration file are needed, please share them.

https://raw.githubusercontent.com/protocolbuffers/protobuf/aa3bc05d2687c87c8dbf35b7b2ebf2a1065be0b1/examples/addressbook.proto

```
$ protoc --cpp_out=. addressbook.proto
```

Reference

-

mikutas committed:
feat: add protocolbuffers/protobuf

[protocolbuffers/protobuf](https://github.com/protocolbuffers/protobuf): Protocol Buffers - Google's data interchange format

```console
$ aqua g -i protocolbuffers/protobuf
```

## How to confirm if this package works well

Reviewers aren't necessarily familiar with this package, so please describe how to confirm if this package works well.
Please confirm if this package works well yourself as much as possible.

Command and output

```console
$
```

If files such as configuration file are needed, please share them.

```
```

Reference

-

suzuki-shunsuke commented:
Thank you for your contribution!

suzuki-shunsuke commented:
I'm not familiar with Protocol Buffer, but I'd like to rename the package to `protocolbuffers/protobuf/protoc` to clarify the package content.
I'll update the pull request.

suzuki-shunsuke committed:
fix: rename protocolbuffers/protobuf to protocolbuffers/protobuf/protoc

To clarify the package content.

https://github.com/aquaproj/aqua-registry/pull/6998#issuecomment-1276014769

suzuki-shunsuke commented:
Released. https://github.com/aquaproj/aqua-registry/releases/tag/v3.74.0

suzuki-shunsuke commented:
Close #1135
